Obsidian/WorldData/RegionFile.cs

```diff
@@ -233,7 +233,7 @@ protected virtual void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-                    FlushToDiskAsync();
+                    Task.Run(async () => await FlushToDiskAsync());
```

Comment:
Should've used [IAsyncDisposable](https://docs.microsoft.com/en-us/dotnet/api/system.iasyncdisposable) in the first place.

Reply:
I have implemented IAsyncDisposable and made the changes to the dispose method